scripts/linter.py

```diff
@@ -0,0 +1,44 @@
+import os
+import re
+from typing import Pattern, List
+
+search_note_pattern: Pattern = re.compile(r'^\*\*Note\*\*')
+search_warning_pattern: Pattern = re.compile(r'^\*\*Warning\*\*')
+correct_note_string: str = '> **Note**'
+correct_warning_string: str = '> **Warning**'
```

Comment:
Since these are constants, can you make them all-caps?

Reply:
Done.